src/util/logger.ts

```diff
@@ -1,11 +1,18 @@
 // eslint-disable-next-line import/named
 import { Logger } from "@zeplin/cli";
 
-let loggerInstance: Logger;
+let loggerInstance: Logger = {
+    error: console.error,
+    info: console.info,
+    warn: console.warn,
+    debug: console.debug
+};
 
-const getLogger = (): Logger | undefined => loggerInstance;
-const setLogger = (logger: Logger): void => {
-    loggerInstance = logger;
+const getLogger = (): Logger => loggerInstance;
+const setLogger = (logger?: Logger): void => {
```

Comment:
I'm confused about the change on `setLogger` function. Why do we need this?

Reply:
Older versions of @zeplin/cli do not pass logger instance to plugins. If the user has updated version of storybook plugin but outdated @zeplin/cli, it would cause `loggerInstance` to set to undefined. Now, it will use `console` functions on that case.

Reply:
I thought this function is called somewhere in `cli-connect-storybook-plugin`. That's why it looks odd to me. I get it now. It makes sense.

Reply:
It is invoked in the plugin's `init` method. That's where CLI pass the logger instance into plugins. This file is just a container